sway-ast/src/priv_prelude.rs

```diff
@@ -35,7 +35,7 @@ pub use {
     },
     extension_trait::extension_trait,
     num_bigint::BigUint,
-    serde::Serialize,
+    serde::{Deserialize, Serialize},
```

Comment:
since theres many serialization formats; and serde is quite common; unsure if we need to re-export this

Reply:
Good point, I guess this PR just re-exports `Deserialize` and `Serialize` was already exported. Should be fairly easy to remove this in the future if `sway-ast` needs to support multiple serialisation methods.